web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -42,13 +42,14 @@ const BookingModalForm = props => {
 
   const composeErrorMessage = () => {
     const { eventTypeId, start } = formData;
-    if (isEventBeingUpdated || eventTypeId !== 1) {
+    const annualLeave = 1;
+    if (isEventBeingUpdated || eventTypeId !== annualLeave) {
```

Comment:
Same as William's comment above, we have constants for these

Reply:
@BrownMark  - rookie mistake. this has been updated